src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+use src\oop\app\src\Parsers\ParserInterface;
+use Symfony\Component\DomCrawler\Crawler;
+
+class KinoukrDomCrawlerParserAdapter implements ParserInterface
+{
+    private Movie $movie;
+
+    public function __construct(Movie $movie)
+    {
+        $this->movie = $movie;
+    }
+
+    /**
+     * @param string $siteContent
+     * @return Movie
+     */
+    public function parseContent(string $siteContent): Movie
+    {
+        $crawler = new Crawler($siteContent);
+        $crawler = $crawler->filter('#dle-content');
+        
+        return $this->movie->setTitle($crawler->filter('h1')->first()->text())
+            ->setPoster("https://kinoukr.com/". $crawler->filter('img')->first()->attr("src"))
```

Comment:
set methods should not have logic inside, please move value calculation into separate variable.

Reply:
done